microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -534,6 +534,36 @@ To enhance the Kids Mode experience, Microsoft Edge adds a safe keyword to the h
 Kids Mode does not collect data for personalization of the news feed or other Microsoft services.  You may not change the privacy settings for Kids Mode.  Other settings like Windows Defender SmartScreen and diagnostic data are configured according to the profile in which Kids Mode was launched.  For more information about diagnostic data about how you use the browser and Windows Defender SmartScreen, see the [Diagnostic Data](#diagnostic-data) and [SmartScreen](#smartscreen) sections.
 
 
+<!-- ====================================================================== -->
+## Magnify Image<!-- todo: is the menuitem capital I or lowercase?  global in section -->
+
+To use the Magnify Image feature, right-click an image and then select **Magnify Image**, or press the **Ctrl** key twice while hovering over an image.  The Magnify Image feature retrieves the largest available version of the image from the website and displays it in a popup window.
```

Comment:
is the menuitem "Magnify image**, or **Magnify Image**? 
trend is toward Sentence case, not Title Case
@sjdallst2 

rendered article section:
https://review.learn.microsoft.com/microsoft-edge/privacy-whitepaper/?branch=pr-en-us-3234#magnify-image

Reply:
The menu item is Magnify image, so this should be fixed.

Reply:
i changed 1 instance:
right-click an image and then select **Magnify image**